### PR TITLE
[copy-quay] Do not push tag if it already exists on destination namespace

### PR DESCRIPTION
**Pull request by michaeltcoelho**

### CODE REVIEW

File: roles/copy_container/files/copy-quay/copy.go [hunk 1]
```diff
@@ -43,6 +43,10 @@ func (opts *copyOptions) run(cmd *cobra.Command, args []string) error {
                 tagToPush = opts.global.pushHash
             }
 
+            if (tagExist(opts.global.toNamespace, image, tagToPush)) {
```
[the remark this answers is not in the document]
Comment (michaeltcoelho) [reply roles/copy_container/files/copy-quay/copy.go:46]: @viroel It seems like I have only considered the human-readable tag in this case. I'll look into the hash one.

**Comment by michaeltcoelho** [comment]
@pablintino I've created new PR containing only code changes #1253

**Comment by lewisdenny** [comment]
Hi @michaeltcoelho will you update this PR to apply formatting to https://github.com/openstack-k8s-operators/ci-framework/pull/1253 ?

**Comment by michaeltcoelho** [comment]
> Hi @michaeltcoelho will you update this PR to apply formatting to #1253 ? If so please update title, description and commit message.

@pablintino I'll have it formatted in a new PR.

**Pull request closed without merging**